suite/runsuite_wrapper.pl

```diff
@@ -231,6 +231,48 @@
                 'code_api|client.nudge_test' => 1, # i#2978
                 'code_api|client.nudge_ex' => 1,
                 'code_api|client.alloc-noreset' => 1, # i#4436
+                # These are from the long suite.
+                'code_api,opt_speed|common.decode-stress' => 1, # i#4845
+                'code_api,thread_private|common.decode-stress' => 1, # i#4845
+                'code_api,thread_private,disable_traces|common.decode-stress' => 1, # i#4845
+                'prof_pcs|common.broadfun' => 1, # i#4845
+                'prof_pcs|common.fib' => 1, # i#4845
+                'prof_pcs|common.decode-bad' => 1, # i#4845
+                'prof_pcs|common.decode' => 1, # i#4845
+                'prof_pcs|common.decode-stress' => 1, # i#4845
+                'prof_pcs|common.eflags' => 1, # i#4845
+                'prof_pcs|common.floatpc' => 1, # i#4845
+                'prof_pcs|common.floatpc_xl8all' => 1, # i#4845
+                'prof_pcs|common.getretaddr' => 1, # i#4845
+                'prof_pcs|common.nativeexec' => 1, # i#4845
+                'prof_pcs|common.protect-dstack' => 1, # i#4845
+                'prof_pcs|common.segfault' => 1, # i#4845
```

Comment:
This looks like the entire set of all prof_pcs tests, with the #1807 limit of `^common`.  Should we instead remove Windows from all prof_pcs runs up front (seeing thread_private below also all failing) and file an issue (if there's not one already) on prof_pcs being broken on WIndows, rather than having every single test on a flaky test list?

Reply:
`prof_pcs|common.logstderr` and `prof_pcs,thread_private|common.logstderr` do pass currently. Do we want them enabled?

Reply:
Oh, so it's not all of them?  That is interesting.  But common.logstderr runs common.broadfun, so how does it pass but common.broadfun fails??   Strange.  I guess it could go either way.  If it's just the one I would still lean toward disabling the whole group.

Reply:
yeah, I disabled the whole group. Just testing now to make sure my changes were correct. Will resolve this comment and re-request review when done.

Reply:
Done.